lib/content-services/src/lib/content-node-selector/content-node-selector-panel/content-node-selector-panel.component.html

```diff
@@ -13,18 +13,26 @@
             adf-auto-focus
             data-automation-id="content-node-selector-search-input">
 
-        <mat-icon *ngIf="searchTerm.length > 0"
-            matSuffix (click)="clear()"
-            class="adf-content-node-selector-content-input-icon"
-            data-automation-id="content-node-selector-search-clear">clear
-        </mat-icon>
+        <button
+            matSuffix
+            mat-icon-button
+            *ngIf="searchTerm.length > 0"
+            data-automation-id="content-node-selector-search-clear"
+            class="adf-content-node-selector-search-clear-button"
+            (click)="clear()"
```

Comment:
Because you moved click action from icon to button - can you correct existing test (or write new if doesn't exist) which checks if clicking on that button works?